bindings/protocols/modbus/ontology.html

```diff
@@ -33,8 +33,8 @@ <h2>Introduction</h2>
         </section>
 
         <section><h2>Axiomatization</h2><section><h3>Classes</h3><section><h4>Data request</h4><p>IRI: <code>https://www.w3.org/2019/wot/modbus#DataRequest</code></p><span>Data request contains additional information sent from a client to server that the server uses to take the action defined by the function code. 
-This can include items like discree and register address, the quantity of items to be handled, and the count of actual data bytes in the field.
-The data request field my be nonexistent (zero length) in certain kid of requests, in this case the server does not require any additional information. The function code alone specifies the action.</span><table class="def"><tbody><tr><td>Sub-class of</td><td><code><a href="#data-field">Data Field</a></code></td></tr></tbody></table></section>
+This can include items like discree?? and register address, the quantity of items to be handled, and the count of actual data bytes in the field.

```

Comment:
```suggestion
This can include items like register address, the quantity of items to be handled, and the count of actual data bytes in the field.
```

I'd remove it, I can't recall what is about.